apps/web/src/components/communication/HealthcareCommunicationSystem.tsx

```
@@ -0,0 +1,1035 @@
+          <div className='flex-1 overflow-y-auto'>
+            {filteredContacts.map(contact => (
+              <div
+                key={contact.id}
+                className={cn(
+                  'flex items-center gap-3 p-4 hover:bg-slate-50 cursor-pointer border-b border-slate-100',
+                  selectedContact === contact.id && 'bg-blue-50 border-blue-200',
+                )}
+                onClick={() => handleContactSelect(contact.id)}
+                role='button'
+                tabIndex={0}
+                onKeyDown={e => {
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)